test/plugin/scenarios/webflux-scenario/pom.xml

```diff
@@ -29,6 +29,7 @@
         <module>webflux-projectA-scenario</module>
         <module>webflux-projectB-scenario</module>
         <module>webflux-dist</module>
+        <module>webflux-webclient-scenario</module>
```

Comment:
@vcjmhg swap this line and line 31 solves the problem,

Reply:
> The newly-added module is not packaged before `webflux-dist` module, so the error message occurred, see my comment inline, please also note that there is another failure in the C

Another failure in the CI is the command -`./mvnw --batch-mode clean package -DskipTests -Pagent`.I have test it in my pc and it has pased.